README.md

```diff
@@ -25,12 +25,26 @@ If u just wanna use the note cards of IELTS words, follow the steps below:
 
 ### Develop
 
-1. Install `npm` or `pnpm`.
+There're actually two ways of generating notes to ur Anki program: 
+- `pnpm note` for converting HTML files and directly importing notes to ur Anki program.
+- `pnpm note-mkanki` is what we **HIGHLY RECOMMEND** for converting HTML files into `apkg`.
+
+#### For `pnpm note`: 
+
+1. Install `npm` or `pnpm`, here we recommend `pnpm`.
 2. `npm install` or `pnpm install` to pull all dependencies.
-3. `pnpm start` to generate html files from Cam dict.
+3. `pnpm start` to generate HTML files from Cam dict.
 4. Open ur Anki program.
-4. Install the Anki plugin [AnkiConnect](https://ankiweb.net/shared/info/2055492159).
-5. `pnpm note` to convert html files to notes in your Anki.
+5. Install the Anki plugin [AnkiConnect](https://ankiweb.net/shared/info/2055492159).
+6. `pnpm note` to convert html files to notes in your Anki.
+
+#### For `pnpm note-mkanki`: 
```

Comment:
> install mkanki which might cost them hours to set up and take up their dick space more or less 10GB...

Does `mkanki` take up 10 GB? 😰 I actually don't know about it. Can u plz explain that more precisely? like which part of it?

Reply:
> > install mkanki which might cost them hours to set up and take up their dick space more or less 10GB...
> 
> Does `mkanki` take up 10 GB? 😰 I actually don't know about it. Can u plz explain that more precisely? like which part of it?

I mean this guy: 

![image](https://github.com/Quasimurdock/IELTS-Anki-Cards-Generator/assets/80401151/fed65bad-fa2b-4813-bc74-dc0b5eab550e)

The components I needed were installed before, so the 'Total space required'(at right bottom) is 0 now. I remember during the components selection the size info displayed was once up to 10 GB. Then I unselected some component, not remember how large they were, the final components should be several GBs.

I checked the Microsoft Visual Studio folder, it's 3.x GB:

![image](https://github.com/Quasimurdock/IELTS-Anki-Cards-Generator/assets/80401151/89bbf888-2815-497b-9185-fbe71496f42d)

Considering some installers were auto deleted after the installation, and some `C:\` files were not included in, the total size of this installation may be close to 10 GB.

Why did I install these things? Because I failed to install mkanki, and the error info showed it's relevant to node-gyp. Then I searched, found a [answer](https://stackoverflow.com/questions/33896511/npm-install-fails-with-node-gyp), the answer said I should set up my environment. Then I read node-gyp's [doc](https://github.com/nodejs/node-gyp#on-windows), installed Python and Visual C++ Build Environment. 

I don't know if it's necessary, but it did work. After these installation, I installed `mkanki` successfully. Maybe you have set up these things before, so you don't need do such things this time. But not every user's environment is ready to install `mkanki`, if they want to install it, they need to spend some time and disk space which is not OK to everyone of them. 

Therefore, I think we should make it clear to users that if they only want to use Anki Connect, they don't need to pay for something(time, disk space) to deal with `mkanki`.

Reply:
Actually it's not a good choice for frontend proj to use VS instead of VSCode. I have never installed previous stuff actually. the only dependency for mkanki are actually npm, pnpm and node.js. I haven't met such environment problems

Reply:
> Actually it's not a good choice for frontend proj to use VS instead of VSCode.

Yeah, VS is just for setting up environment.

> but I think we shouldn't assume those who would pull this proj and run locally don't know about development

At least we can give suggestions, it may save their time.

Reply:
> the only dependency for mkanki are actually npm, pnpm and node.js.

I've already get all of them installed, Node.js and npm are latest LTS version, pnpm is latest version, but I do encounter the mkanki installation failure which I mentioned above at that time...

Reply:
Yeah that's weird. Cuz I actually set up the proj on two different PC and didn't encounter that. Well anyway, u can keep that note in README

Reply:
> Yeah that's weird. Cuz I actually set up the proj on two different PC and didn't encounter that. Well anyway, u can keep that note in README

Maybe your PCs are all get Python installed before. If it's so, I'll uninstall Visual C++ Build Environment, only keep Python, then try to install mkanki to see whether the installation is successful.